build/dependencies.props

```diff
@@ -14,7 +14,7 @@
     <HealthCheckAWSSystemsManager>6.0.0</HealthCheckAWSSystemsManager>
     <HealthCheckAzureDigitalTwin>6.0.0</HealthCheckAzureDigitalTwin>
     <HealthCheckAzureServiceBus>6.0.4</HealthCheckAzureServiceBus>
-    <HealthCheckAzureStorage>6.0.4</HealthCheckAzureStorage>
+    <HealthCheckAzureStorage>6.0.5</HealthCheckAzureStorage>
```

Comment:
New APIs may warrant a minor version bump. Although, perhaps we could re-use an existing API

Reply:
I'm not sure about versioning model here (patch or minor). ping @carlosrecuero

Reply:
Personally I'm fine either way.

Reply:
I've bumped it to minor for now, but I noticed that no other package has done so haha

Reply:
I don't know intents of maintainers of this repo regarding minor/patch but I know that each package can be published independently with its own version.

Reply:
Ah ok. Then probably minor is the most semantically appropriate here